### DISCUSSION TITLE
Importing SVGs into Friction

### DISCUSSION THREAD

**Discussion by pgilfernandez** [friction2d/friction#308] [Q&A]
I've been playing importing assets from Inkscape to Friction and the general feeling is great, they import perfectly, fast and colors and lots of elements are transferred without any issue, great.

The only thing that annoys me a little bit and I think could be improved (from the POV of a newbie or general user with Inkscape background/knowledge) is that the name of elements (layers, groups) imported into Friction are taken from the IDs of the Inkscape elements and not from the labels... I think users are more used to name their elements using the outliner (layers panel) than opening the Properties editor and filling the ID field.
It is just an opinion but I would say that people would find more UX logical to use the labels instead of IDs... I know that the point is not to have elements with the same name (forbidden in Friction) but at import time they could be renamed with an incremental number or so... Or well, add a Preference option to allow users to choose the behavior.

It's not something very important, but well, I would like to drop my little stone

Cheers

**Comment by rodlie** [comment]
Sure, can probably look into that (I need to fix compat with Affinity SVG's anyway). The SVG importer is generic, so Inkscape (or any other app) specific stuff are ignored.

**Reply by rodlie** [reply]
Also see #185

**Comment by rodlie** [comment]
Fixed in https://github.com/friction2d/friction/commit/f72fc9906d049c99c09e135a473fa1f01343cc90

Will be merged into main sometime tomorrow.

**Comment by NusryNizam** [comment]
This is awesome. I had the exact same issue, and I come here. It's fixed. Wow.
